<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>720</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>XAI Viz</string>
  </property>
  <widget class="QComboBox" name="choose_layer">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>45</y>
     <width>380</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>50</y>
     <width>55</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Layer</string>
   </property>
  </widget>
  <widget class="QPushButton" name="back_btn">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>30</y>
     <width>60</width>
     <height>60</height>
    </rect>
   </property>
   <property name="text">
    <string>Back</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="target_choice">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>630</y>
     <width>111</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Direction Loss</string>
   </property>
  </widget>
  <widget class="QPushButton" name="generate_btn">
   <property name="geometry">
    <rect>
     <x>1090</x>
     <y>620</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Generate</string>
   </property>
  </widget>
  <widget class="QPushButton" name="save_btn">
   <property name="geometry">
    <rect>
     <x>1090</x>
     <y>660</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Export Image</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="grad_cam">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>660</y>
     <width>111</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Grad-Cam</string>
   </property>
  </widget>
  <widget class="QLabel" name="filter">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>580</y>
     <width>100</width>
     <height>100</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
